Checkout is not active when product options are not chosen

MarekB01 opened the pull request:
https://github.com/BroadleafCommerce/QA/issues/4765

Added new method to Order, so it can reference cart and render it in html

MarekB01 committed:
BroadleafCommerce/QA-4765_checkout_is_not_active_if_product_options_not_chosen
Added new method to Order, so it can reference cart and render it in html

oleksiimiroshnyk commented on core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/order/domain/NullOrderImpl.java:
```diff
@@ -425,4 +425,9 @@ public <G extends Order> CreateResponse<G> createOrRetrieveCopyInstance(MultiTen
+
+    @Override
+    public Boolean hasValidationErrors() {
+        return null;
```
do we need here null of false ?

MarekB01 committed:
BroadleafCommerce/QA-4765_checkout_is_not_active_if_product_options_not_chosen
changed return statement on false instead of null in hasValidationErrors method in NullOrderImpl
